jbpm-db-scripts/src/test/java/org/jbpm/persistence/scripts/DDLScriptsTest.java

```diff
@@ -16,28 +16,79 @@
-import static org.jbpm.persistence.scripts.TestPersistenceContext.createAndInitContext;
+import static org.jbpm.test.persistence.scripts.util.ScriptFilter.filter;
+import static org.junit.Assume.assumeTrue;
 
 /**
  * Contains tests that test DDL scripts.
  */
+@RunWith(Parameterized.class)
 public class DDLScriptsTest extends ScriptsBase {
+
+    @Parameters
+    public static Collection<ScriptFilter[]> data() {
+        ScriptFilter[] standard = new ScriptFilter[]{ScriptFilter.init(false, true), ScriptFilter.init(false, false)};
+
+        ScriptFilter[] springBoot = new ScriptFilter[]{ScriptFilter.init(true, true)
+                                                                   .setSupportedDatabase(DatabaseType.POSTGRESQL, DatabaseType.ORACLE),
+                                                       ScriptFilter.init(true, false)};
+
+
+        ScriptFilter[] pqlBytea = new ScriptFilter[]{filter("postgresql-bytea-jbpm-schema.sql")
+                                                                 .setSupportedDatabase(DatabaseType.POSTGRESQL)
+                                                                 .setOptions(Option.DISALLOW_EMTPY_RESULTS, Option.THROW_ON_SCRIPT_ERROR),
+                                                     filter("postgresql-bytea-jbpm-schema-drop.sql")};
+
+        ScriptFilter[] pqlSpringBootBytea = new ScriptFilter[]{filter("postgresql-springboot-bytea-jbpm-schema.sql")
+                                                                           .setSupportedDatabase(DatabaseType.POSTGRESQL)
+                                                                           .setOptions(Option.DISALLOW_EMTPY_RESULTS, Option.THROW_ON_SCRIPT_ERROR),
+                                                               filter("postgresql-springboot-bytea-jbpm-schema-drop.sql")};
+
+        return Arrays.asList(standard, springBoot, pqlBytea, pqlSpringBootBytea);
+    }
+
+    private ScriptFilter createScript;
+    private ScriptFilter dropScript;
+
+    public DDLScriptsTest(ScriptFilter createScript, ScriptFilter dropScript) {
+        this.createScript = createScript;
+        this.dropScript = dropScript;
+    }
+
+    @Before
+    public void prepare() {
+        TestPersistenceContextBase scriptRunnerContext = createAndInitContext(PersistenceUnit.SCRIPT_RUNNER);
+        DatabaseType dbType = scriptRunnerContext.getDatabaseType();
+        assumeTrue("Scripts test not supported this database " + dbType, createScript.isSupportedDatabase(dbType));
+    }
     /**
      * Tests that DB schema is created properly using DDL scripts.
      */
     @Test
     public void createAndDropSchemaUsingDDLs() throws Exception {
-        executeScriptRunner(DB_DDL_SCRIPTS_RESOURCE_PATH, ScriptFilter.init(false, true));
+        executeScriptRunner(DB_DDL_SCRIPTS_RESOURCE_PATH, createScript);
         validateAndPersistProcess();
         validateQuartz();
```

Comment:
This quartz validation is failing for non-standard configurations (i.e., 1, 2 and 3 of parameterized configs for postgres and 1 for oracle):

For these configurations (e.g. [1]):
_Returned DDL files: [postgresql-springboot-jbpm-schema.sql]_

instead of working configurations [0]:

_Returned DDL files: [postgresql-jbpm-schema.sql, postgresql-jbpm-lo-trigger-clob.sql, quartz_tables_postgres.sql, task_assigning_tables_postgresql.sql]_

At least, quartz_tables_postgres.sql should be either returned for all configurations or skipping this validations for all configurations but 0.